Project.toml

```
@@ -1,9 +1,12 @@
 name = "HDF5"
 uuid = "f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
-version = "0.15.5"
+version = "0.16.0"
```

```suggestion
version = "0.15.5"
```

I prefer to bump the version in a separate PR

---

Sure, although I would like to see this sooner rather than later. It's rather difficult to develop against the master branch at the moment because the version is lagging the release, yet there are major API changes (such as the new API module).

---

Could you explain why it is difficult to develop against the master branch? AFAIK you can just checkout the master branch and work off of it, which is independent of the release version.  There's always going to be a lag between release versions and development versions.

---

I have some projects which depend on the released version, and I have other projects that depend on the master branch.

The projects which depend on the 0.15.x release have `const API = HDF5`. If I want to test those projects against master, I have to manually change that line for all modules that import HDF5 because I cannot automatically change this based on version number of HDF5.jl.

---

Makes sense. Done in https://github.com/JuliaIO/HDF5.jl/blob/master/Project.toml

The registration happens on a specific commit, right, and not on the commit that last updated the `Project.toml` file. So I guess it makes sense to change the version number on `Project.toml` even if we don't have a final release ready.

---

Since this is already changed on master, can I leave this here?